src/routes/docs/+page.svelte

```diff
@@ -35,7 +35,7 @@
             href: '/docs/tutorials/vue',
             cover: {
                 src: '/images/tutorials/vue.png',
-                alt: 'Vue logo'
+                alt: ''
```

Comment:
If we never want to have the alt in these images, we probably should not expect it as a prop in `CodeCard`